index.js

```diff
@@ -0,0 +1,565 @@
+
+const ngrok = "https://74b76b4a.ngrok.io/";
+const filepath = "/Users/ek/Projects/dasmitdersilbe/";
```

Comment:
No paths should be hard-coded.

Reply:
ngrok will be used only for development purposes
I delete the filepath variable in the next version